privacy-policy.md

```diff
@@ -72,6 +76,10 @@ b. *Service Providers*.
 We share User Information with our trusted third party service providers and other individuals who perform services on our behalf, for example, providing customer service support, hosting services, analytics and other services we utilize to help us provide our Service or conduct our business.
 These service providers access and use User Information only to provide services to Desmos under reasonable confidentiality terms.
 
+We share Personally Identifiable Information with third parties only if such providers have a need to access such information for the purpose of carrying out their work for us and do so under strict confidentiality and data security terms.
+
+A list of Third Party Services can be found [here](https://github.com/desmosinc/policies).
```

Comment:
This should link to https://github.com/desmosinc/policies/blob/master/third-party-services.md